site/content/docs/5.3/components/toasts.md

```diff
@@ -95,7 +95,7 @@ We use the following JavaScript to trigger our live toast demo:
 
 Toasts are slightly translucent to blend in with what's below them.
 
-{{< example class="bg-dark" stackblitz_add_js="true" >}}
+{{< example class="bg-secondary" stackblitz_add_js="true" >}}
```

Comment:
We shouldn't use `.bg-secondary` here as it switches off to a white background in dark mode which doesn't exist on our side. To still show the feature, `.bg-body` doesn't work?

Reply:
The goal here is to show that the toast-body is transparent, I feel that with `.bg-secondary` we really see it well.